LoginApplication/app/src/main/java/com/example/webonise/loginapplication/SignUpActivity.java

```diff
@@ -0,0 +1,281 @@
+
+        btnLogin = (Button) findViewById(R.id.btnLogin);
+        btnCancel = (Button) findViewById(R.id.btnCancel);
+
+        input_layout_fname = (TextInputLayout) findViewById(R.id.input_layout_fname);
+        input_layout_lname = (TextInputLayout) findViewById(R.id.input_layout_lname);
+        input_layout_contact = (TextInputLayout) findViewById(R.id.input_layout_contact);
+        input_layout_email = (TextInputLayout) findViewById(R.id.input_layout_email);
+        input_layout_password = (TextInputLayout) findViewById(R.id.input_layout_password);
+        input_layout_confirmPassword = (TextInputLayout) findViewById(R.id.input_layout_cpassword);
+
+        radioGroup.setOnCheckedChangeListener(this);
+        spinner.setOnItemSelectedListener(this);
+        btnLogin.setOnClickListener(this);
+        btnCancel.setOnClickListener(this);
+
+        List<String> itemSpinner = new ArrayList<>();
+        itemSpinner.add("What is your nick name?");
+        itemSpinner.add("Who is your favourite actor?");
+        itemSpinner.add("Which school you went?");
+        itemSpinner.add("First pet you had?");
+        // Creating adapter for spinner
+        ArrayAdapter<String> dataAdapter = new ArrayAdapter<String>(this, android.R.layout.simple_spinner_item, itemSpinner);
+        // Drop down layout style - list view with radio button
+        dataAdapter.setDropDownViewResource(android.R.layout.simple_spinner_dropdown_item);
+        // attaching data adapter to spinner
+        spinner.setAdapter(dataAdapter);
+    }
+
+    @Override
+    public void onItemSelected(AdapterView<?> parent, View view, int position, long id) {
+        String item = parent.getItemAtPosition(position).toString();
+    }
+
+    @Override
+    public void onNothingSelected(AdapterView<?> parent) {
+    }
+
+    @Override
+    public void onClick(View v) {
+
+        switch (v.getId()) {
+            case R.id.btnLogin:
+                onLoginButtonClicked();
+                break;
+            case R.id.btnCancel:
+                System.exit(0);
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void onLoginButtonClicked() {
+
+        firstname = editFirstName.getText().toString();
+        lastname = editLastName.getText().toString();
+        contactString = editContact.getText().toString();
+        email = editEmail.getText().toString();
+        password = editPassword.getText().toString();
+        confirmPassword = editConfirmPassword.getText().toString();
+        address = editAddress.getText().toString();
+        securityQuestion = spinner.getSelectedItem().toString();
+        securityAnswer = editAnswer.getText().toString();
+
+        contactdouble = Double.parseDouble(contactString);
+        contact = (int) contactdouble;
+
+        if (!valiDateProfile()) {
+            Toast toast = Toast.makeText(getApplicationContext(), "Enter Data", Toast.LENGTH_SHORT);
```

Comment:
Shouldn't use hardcoded string values. Take them out in strings.xml and use reference